libs/datasets/sources/cdc_vaccine_dataset.py

```diff
@@ -49,4 +49,11 @@ class CDCVaccinesDataset(data_source.CanScraperBase):
             provider="cdc",
             common_field=CommonFields.VACCINATIONS_ADDITIONAL_DOSE,
         ),
+        ccd_helpers.ScraperVariable(
+            variable_name="total_vaccine_bivalent_dose",
+            measurement="cumulative",
+            unit="people",
+            provider="cdc",
+            common_field=CommonFields.VACCINATIONS_BIVALENT_DOSE,
+        ),
```

Comment:
I don't quite remember the context of cdc_vaccine_dataset.py and cdc_new_ vaccine_dataset.py, do we need this in both?

Reply:
At this point these could be merged, but `CDCVaccinesDataset` gives us state-level data and the other gives us county-level. 

We would need to make a small change in `can-scrapers` to change the county-level dataset source name, then I think we could merge them, but I don't 100% remember why they were split, so maybe there's more there.

Reply:
I've added comments explaining this for now

Reply:
Ah, right.  That makes sense.  Thanks for adding comments.  No need to rework.